python/ray/worker.py

```diff
@@ -434,9 +435,15 @@ def store_with_plasma(self, object_id, value):
                     value, object_id, memcopy_threads=self.memcopy_threads)
             else:
                 writer = Pickle5Writer()
-                meta = pickle.dumps(
-                    value, protocol=5, buffer_callback=writer.buffer_callback)
-                self.core_worker.put_pickle5_buffers(object_id, meta, writer,
+                try:
+                    inband = native_pickle.dumps(
```

Comment:
We need to keep using cloudpickle here, right?

Reply:
I think we should first try pickle, if it fails, then fallback to cloudpickle. Just like PyArrow.

Reply:
Wait a minute. Let me test the overhead of cloudpickle_fast again.

Reply:
Yeah, let's try to use cloudpickle throughout, I think there shouldn't be any performance problems with cloudpickle_fast :)

Reply:
yep, you are right. I have updated it. Should we make `USE_NEW_SERIALIZER` default for python3.8?

Reply:
I think we should have a flag that can be passed into `ray.init()` as in https://github.com/ray-project/ray/pull/5805/files and then switch it on by default once we have done more testing :) I can finish that PR and try to get it merged soon.

Do you want to start working on making cloudpickle_fast work with the pickle5 backport?

Reply:
sure, I would like to improve that